src/pages/docs/chat/getting-started/kotlin.mdx

```diff
@@ -0,0 +1,1040 @@
+* Disconnecting and resource cleanup
+
+## Prerequisites <a id="prerequisites"/>
+
+### Ably <a id="prerequisites-ably"/>
+
+1. Sign up for an Ably account.
+2. Create a new app and get your API key. You can use the root API key that is provided by default to get started.
+
+3. Install the Ably CLI:
+
+<Code>
+```shell
+npm install -g @ably/cli
+```
+</Code>
+
+4. Run the following to log in to your Ably account and set the default app and API key:
+
+<Code>
+```shell
+ably login
+
+ably apps switch
+ably auth keys switch
+```
+</Code>
+
+<Aside data-type='note'>
+The code examples in this guide include a demo API key. If you wish to interact with the Ably CLI, ensure that you replace them with your own API key.
+</Aside>
+
+### Create a new project <a id="prerequisites-project"/>
+
+Create a new Android project with Jetpack Compose. For detailed instructions, refer to the [Android Studio documentation](https://developer.android.com/jetpack/compose/setup).
+
+1. Create a new Android project in Android Studio.
+2. Select **Empty Activity** as the template
+3. Name the project **Chat Example** and place it in the `com.example.chatexample` package
+4. Set the minimum SDK level to API 24 or higher
+5. Select Kotlin as the programming language
+6. Add the Ably dependencies to your app-level `build.gradle.kts` file:
+
+<Code>
+```kotlin
+implementation("com.ably.chat:chat-android:<latest-version>")
+// This package contains extension functions for better Jetpack Compose integration.
+// It's experimental for now (safe to use, but the API may change later). You can always use its code as a reference.
+implementation("com.ably.chat:chat-extensions-compose:<latest-version>")
+```
+</Code>
+
+## Step 1: Setting up Ably <a id="step-1"/>
+
+Replace the contents of your `MainActivity.kt` file with the following code to set up the Ably client:
+
+<Code>
+```kotlin
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import androidx.compose.foundation.layout.Colum
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import java.util.UUID
+import com.example.chatexample.ui.theme.ChatExampleTheme
+import com.ably.chat.*
+import io.ably.lib.realtime.AblyRealtime
+import io.ably.lib.types.ClientOptions
+
+class MainActivity : ComponentActivity() {
+    private lateinit var realtimeClient: AblyRealtime
+    private lateinit var chatClient: ChatClient
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        realtimeClient = AblyRealtime(
+            ClientOptions().apply {
+                key = "{{API_KEY}}"
```

Comment:
Should we be encouraging any sort of good practice here (e.g. putting the key in config)?

Definitely also worth a warning that you should use tokens in prod